examples/stats/main.go

```diff
@@ -25,17 +25,17 @@ import (
 	"time"
 
 	"github.com/census-instrumentation/opencensus-go/stats"
-	"github.com/census-instrumentation/opencensus-go/tags"
+	"github.com/census-instrumentation/opencensus-go/tag"
```

Comment:
I am not 100% convinced that this is a good thing to do. The reasons are the following:
1) There are packages using plural in the main golang (https://golang.org/pkg/) - bytes, errors, strings, etc.
2) All the other languages are using tags.
3) Using stat instead of "stats" is not a great choice.

Reply:
I am not planning to rename stats to stat. What's represented in stats is almost "stats collection" which sounds different when stats is singularized.

To be fair, you cannot name things after byte, error and string which are builtins in Go. I wouldn't use them as a reference.

I feel the second argument though. I am not convinced that I should merge this change.

Reply:
Given this a second thought, I am now supporting this change because the package name does not have the same importance in other languages the way it has in Go. In other languages, you (or your IDE automatically) import the symbol from a package and your call sites never have to deal with that name. We shouldn't care much about consistency among languages given they do not care about package organization and naming equally.